### PR TITLE
docs: updated README.md and  updated contribution.md

### PR DESCRIPTION
**Pull request by mrunalvilas**
docs: updated README.md and docs: updated contribution.md

### CODE REVIEW

File: contribution.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+```terminal
+  git add .
+  git commit -m "<your_commit_message>"
+  
+  # If so try to use conventional commit messages using the guide: https://www.conventionalcommits.org/en/v1.0.0/
+```
+
+**11.** Push your local branch to the remote repository.
+```terminal
+git push
```
Comment (dwvicy) [inline contribution.md]: git push origin <branch-name>

File: contribution.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to dscrait.tech? You can start by looking through these `good first issues`:
+
+* Good First issues - issues which should only require a few lines of code, and a test or two.
+
+## Styleguides
+
+### Git Commit Messages
```
Comment (dwvicy) [inline contribution.md]: add commit message conventions are already added above: https://www.conventionalcommits.org/en/v1.0.0/

File: README.md [hunk 3]
```diff
@@ -1,5 +1,114 @@
-# dscrait.tech
+
+<h1 align="center"> dscrait.tech README</h1>
+
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/dscrait/dscrait.tech">
+    <img src="https://raw.githubusercontent.com/mrunalvilas/dscrait.tech/update_readme_docs/logo/DSCLOGO.PNG" width="300" height="300">
```
Comment (dwvicy) [inline README.md]: stretch out the logo or remove it

File: contribution.md (hunk 1 above)
Comment (mrunalvilas) [reply contribution.md]: Okay I will make the required changes, let me know if any thing else to change!!

File: README.md [hunk 4]
```diff
@@ -1,5 +1,114 @@
...
 DSC-RAIT is a community of budding developers, entrepreneurs, and innovators, getting ready to solve real-world problems. Our mission is to inculcate an innovative mindset in our members to help our local community as well as contributing towards solving global crises. We hope to build impactful software and guide the future leaders of our world.
+
+
+### Built With
+Major frameworks that are used to  built this project are listed below:
+* [Typescript](https://www.typescriptlang.org/)
+* [NextJS](https://nextjs.org/)
+* [Javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+* [ReactJS](https://reactjs.org/)
+* [NodeJS](https://nodejs.org/en/)
```
Comment (dwvicy) [inline README.md]: no need to add so much stuff. just write built with next and react

File: README.md [hunk 5]
```diff
@@ -1,5 +1,114 @@
...
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<!-- CONTACT -->
+## Contact
+
+DSC-RAIT: [dsc.rait@gmail.com](dsc.rait@gmail.com) 
+
+Project Link: [https://github.com/dscrait/dscrait.tech](https://github.com/dscrait/dscrait.tech)
+
+You can join our Discord server for more details:  [Discord](https://discord.com/channels/743724391296270377/749248351563677697)
```
Comment (dwvicy) [inline README.md]: I don't think this link is an invite link, please change it to the invite link sent to your email when you enrolled

File: README.md [hunk 6]
```diff
@@ -1,5 +1,114 @@
...
+
+
+<!-- ACKNOWLEDGEMENTS -->
+## Acknowledgements
+* [GitHub Emoji Cheat Sheet](https://www.webpagefx.com/tools/emoji-cheat-sheet)
+* [bootstrap-styled/v4](https://bootstrap-styled.github.io/v4/)
+* [commitlint/cli](https://commitlint.js.org/#/)
+* [cheerio](https://cheerio.js.org/)
+* [react-dom](https://reactjs.org/docs/react-dom.html)
+* [styled-components](https://styled-components.com/)
```
Comment (dwvicy) [inline README.md]: remove this section #acknowledgements

File: contribution.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
+# Contributing to dscrait.tech
+
+First off, thanks for taking the time to contribute! 
+
+The following is a set of guidelines for contributing to dscrait.tect, which is hosted in the [dscrait.tech](https://dscrait.tech). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
```
Comment (dwvicy) [inline contribution.md]:
```suggestion
The following is a set of guidelines for contributing to dscrait.tech, which is hosted in the [dscrait.tech](https://dscrait.tech). These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document by opening up an issue.
```

File: contribution.md (hunk 2 above)
Comment (dwvicy) [reply contribution.md]: Remove this section

**Review by dwvicy — CHANGES_REQUESTED** [review]

File: README.md [hunk 8]
```diff
@@ -1,5 +1,89 @@
...
+
+<!-- LICENSE -->
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<!-- CONTACT -->
+## Contact
+
+DSC-RAIT: [dsc.rait@gmail.com](dsc.rait@gmail.com)
```
Comment (dwvicy) [inline README.md]:
```suggestion
- [Email us](dsc.rait@gmail.com) 
```

File: README.md [hunk 9]
```diff
@@ -1,5 +1,89 @@
...
+## License
+
+Distributed under the MIT License. See `LICENSE` for more information.
+
+<!-- CONTACT -->
+## Contact
+
+DSC-RAIT: [dsc.rait@gmail.com](dsc.rait@gmail.com) 
+
+Project Link: [https://github.com/dscrait/dscrait.tech](https://github.com/dscrait/dscrait.tech)
```
Comment (dwvicy) [inline README.md]: Remove the project link

File: README.md [hunk 10]
```diff
@@ -1,5 +1,89 @@
-# dscrait.tech
+
+<h1 align="center"> dscrait.tech README</h1>
```
Comment (dwvicy) [inline README.md]:
```suggestion
<h1 align="center"> DSC-RAIT Website Project</h1>
```

**Review by dwvicy — CHANGES_REQUESTED** [review]

**Comment by meta-boy** [comment]
These many reviews for one readme :'). Also delete the branch as it's not following [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/l) like every other commit on the repo.

**Comment by dwvicy** [comment]
> These many reviews for one readme :'). Also delete the branch as it's not following [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/l) like every other commit on the repo.

It is required, people are contributing to os for the first time, let them learn @meta-boy ¯\_(ツ)_/¯

File: contribution.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+**3.** Navigate to the project directory.
+```terminal
+cd dscrait.tech
+```
+
+**4.** Install NPM packages.
+```terminal
+npm install
```
Comment (dwvicy) [inline contribution.md:70]:
```suggestion
yarn
```

File: contribution.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+```
+**5.** Run the website in your localhost by typing this command.
+```terminal
+npm run dev
```
Comment (dwvicy) [inline contribution.md:74]: yarn run dev

**Review by dwvicy — CHANGES_REQUESTED** [review]

**Review by dwvicy — APPROVED** [review]

**Pull request merged**